datalad/utils.py

```diff
@@ -2430,7 +2551,7 @@ def create_tree(path, tree, archives_leading_dir=True, remove_existing=False):
         os.makedirs(path)
 
     if isinstance(tree, dict):
-        tree = tree.items()
+        tree = list(tree.items())
```

Comment:
pragmatically doesn't matter but in principle -- why is this needed here?  ideally we should be ok with `tree` remaining some iterable instead of populated into a `list`

Reply:
`tree` is annotated in the function signature as a tuple, list, or dict, and the return value of `tree.items()` is none of those, so we have to convert it to one of those.